src/index.js

```diff
@@ -152,6 +152,9 @@ export default {
   get TabController() {
     return require('./components/tabController').default;
   },
+  get TabController2() {
```

Comment:
As discussed, I think the migration process here will be a little different. 
Since the API is completely the same, I'd like to first check the component in OneApp and see that it works fine with no issue (we can change the export in the private lib). 
Later on, if everything works we can just swap the exports in public